inference/generativeai/llm-workshop/llama2-7b-batching-throughput/llama2-7b-batching-throughput.ipynb

```diff
@@ -0,0 +1,1713 @@
+{
```

Comment:
<p>You can download straight from our public bucket at s3://sagemaker-example-files-prod-us-west-2/models/llama-2/fp16/7B/ - this top make sure the notebook still works even if the public model is removed or edited. You can comment the code out to download from the repo for customers to uncomment and run if they wish to do so</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4419/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Added predefined s3 url for llama2-7b